chia/full_node/weight_proof.py

```diff
@@ -650,7 +650,7 @@ def _get_weights_for_sampling(
         q = 1 - delta ** u
         # todo check division and type conversions
         weight = q * float(total_weight)
-        weight_to_check.append(uint128(weight))
+        weight_to_check.append(uint128(int(weight)))
```

Comment:
Why does this one have to be an int?

Reply:
`weight` is a float, and not casting here makes mypy complain